### ISSUE TITLE
useKeyPress trigger while input is focused

### ISSUE THREAD

**Issue by Maquinours** [wbkd/react-flow#3178]
### Please describe the feature that you want to propose

Hi.

In my project, I use a UseKeyPress to trigger when CTRL + A is pressed to select all nodes & edges.
Also, my nodes contains inputs to enter some text.

The problem is, when I use CTRL + A on my input to select all the text inside, the UseKeyPress is triggered so it selects all my nodes.

Can we get a way to avoid useKeyPress to be triggered if a certain element is focused ? I tried the "nopan" className but it doesn't work in this case.

Thank you !

**Comment by moklick** [comment]
did you try to use the class name "nokey" ?

**Comment by Maquinours** [comment]
Yep. I tried "nokey" and it didn't work. Actually, I used "nokey" for months and it was working.
Recently, I noticed that it was not working anymore.

**Comment by moklick** [comment]
could you try: 

```js
const ctrlAPressed = useKeyPress('Control+a', { actInsideInputWithModifier: false });
```

?

**Comment by Pjaerr** [comment]
👋  Thanks @moklick I had the same issue and this suggestion fixed it for me. Is it worth getting this documented somewhere?
